lesson_08/test_functions.py

```diff
@@ -0,0 +1,53 @@
+# Функція на обрізання строки де перший параметр це строка а другий це число скільки символів цієї строки буде виведено
+def my_substr(string, slice):
+    index = 0
+    new_string = ''
+    while index < slice:
+        new_char = string[index]
+        new_string = new_string + new_char
+        index = index + 1
+    return new_string
```

Comment:
Тут можна було попростіше
```suggestion
    return string[:slice]
```